src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -1626,7 +1626,7 @@ private void WriteStringIndented(ReadOnlySpan<byte> escapedPropertyName, ReadOnl
             int indent = Indentation;
             Debug.Assert(indent <= 2 * JsonConstants.MaxWriterDepth);
 
-            Debug.Assert(escapedValue.Length <= JsonConstants.MaxTokenSize);
+            Debug.Assert(escapedValue.Length <= JsonConstants.MaxEscapedTokenSize);
```

Comment:
Should this be `MaxEscapedCharacterTokenSize` instead since this is talking about `ROS<char>`, not `ROS<byte>`?

Reply:
See above.